lib/cisco_node_utils/bgp_af.rb

```diff
@@ -602,11 +635,15 @@ def networks=(should_list)
       [:add, :remove].each do |action|
         Cisco::Logger.debug("networks delta #{@get_args}\n #{action}: " \
                             "#{delta_hash[action]}")
-        delta_hash[action].each do |network, route_map|
+        delta_hash[action].each do |network, route_map_policy|
           state = (action == :add) ? '' : 'no'
           network = Utils.process_network_mask(network)
-          route_map = "route-map #{route_map}" unless route_map.nil?
-          set_args_keys(state: state, network: network, route_map: route_map)
+          unless route_map_policy.nil?
+            route_map = "route-map #{route_map_policy}"
+            route_policy = "route-policy #{route_map_policy}"
+          end
+          set_args_keys(state: state, network: network, route_map: route_map,
+                        route_policy: route_policy)
```

Comment:
The whole route-map vs route-policy behavior (where sometimes we treat them as the same thing and other times we treat them as completely separate) still smells wrong to me. Not blocking for this PR but I'll open a user story to revisit this in general.